PSKoans/Koans/Foundations/AboutAssertions.Koans.ps1

```diff
@@ -1,5 +1,5 @@
 #Requires -Modules PSKoans
-[Koan(Position = 101)]
+[Koan(Position = 101, HelpPath = "about_comparison_operators", HelpURL = "https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comparison_operators?view=powershell-6" )]
```

Comment:
The sole thing I'm not a fan of with these is the long-ass HelpURLs. 

Can we abstract those out? For example... set a variable somewhere with the URL in it and just use that? Or instead of HelpURL we could have `HelpKey` or some such that is the relevant key of a hashtable with the values being these long URLs?

Reply:
Or even just have the file name itself specify the URLs and such... but I'm not sure if that would end up needing to be changed later...